src/app/home-toolbar/home-toolbar.component.html

```diff
@@ -1,9 +1,21 @@
 <md-toolbar color="primary">
-  <span routerLink="home-body">Language Is Fun!</span>
-  <span routerLink="home-manga-browser" style="margin-left:30px;">Read Manga</span>
+  <div>
+ 
+  <button md-button class="md-primary" routerLink="/home" title="home" >
+    Languge is fun!
+  </button>
 
-  <!-- This fills the remaining space of the current row -->
-  <span class="fill-remaining-space"></span>
+  <button md-button class="md-raised" routerLink="/home-manga-browser">
```

Comment:
In general do the same thing as what you removed. This should be routerLink="home-manga-browser". If you have any problems once you go to the user-profile section it could be related to the relative pathing thing, in which case this might need to be routerLink="/home/home-manga-browser".